janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLConfigOptions.java

```diff
@@ -283,6 +284,175 @@
              * the same Cassandra DC.
              */
             ConfigOption.Type.MASKABLE,
-            String.class);
-
+            String.class,
+        "datacenter1");
+
+    // Netty
+
+    ConfigNamespace NETTY = new ConfigNamespace(
+        CQL_NS,
+        "netty",
+        "Configuration options related to the Netty event loop groups used internally by the CQL driver.");
+
+    ConfigOption<Integer> NETTY_IO_SIZE = new ConfigOption<>(
+        NETTY,
+        "io-size",
+        "The number of threads for the event loop group used for I/O operations " +
+            "(reading and writing to Cassandra nodes). " +
+            "If this is set to 0, the driver will use `Runtime.getRuntime().availableProcessors() * 2`.",
+        ConfigOption.Type.LOCAL,
+        Integer.class,
+        0);
+
+    ConfigOption<Integer> NETTY_ADMIN_SIZE = new ConfigOption<>(
+        NETTY,
+        "admin-size",
+        "The number of threads for the event loop group used for admin tasks not related to request I/O " +
+            "(handle cluster events, refresh metadata, schedule reconnections, etc.). " +
+            "If this is set to 0, the driver will use `Runtime.getRuntime().availableProcessors() * 2`.",
+        ConfigOption.Type.LOCAL,
+        Integer.class,
+        0);
+
+    ConfigOption<Long> NETTY_TIMER_TICK_DURATION = new ConfigOption<>(
+        NETTY,
+        "timer-tick-duration",
+        "The timer tick duration in milliseconds. This is how frequent the timer should wake up to check for timed-out tasks " +
+            "or speculative executions. See DataStax Java Driver option " +
```

Comment:
nitpick: missing backquote around `DefaultDriverOption.NETTY_TIMER_TICK_DURATION.getPath()`. Same for next config option.